src/server/methods.ts

```diff
@@ -22,13 +22,25 @@ export const games: { [gameID: number] : Game } = {
   0: new Game(
     // new Map(38, 38, JSON.parse(fs.readFileSync( path.join(__dirname, 'saves/0.json') ).toString()).map),
     // new Map(38, 38, ...new WorldGenerator(3634, 38, 38).generate(0.5, 0.9, 1)),
-    new Map(200, 200, ...new PerlinWorldGenerator(1, 200, 200).generate()),
+    new PerlinWorldGenerator(1, { width: 20, height: 20 }).generate(),
     {
       playerCount: 1,
     }
   ),
 };
 
+const createGame = (username: string, playerCount: number, mapOptions: MapOptions, options: { seed?: number, gameName?: string }) => {
+  const newID = Object.keys(games)[Object.keys(games).length - 1] + 1;
```

Comment:
Potential bug when creating a game after deleting an old game?
For example, if `games` is `{0: Game(...), 1: Game(...), 2: Game(...)}`, and game[1] ends (and should be deleted), resulting in `{0: Game(...), 2: Game(...)}`, then `newID` will be `1`.